superset-frontend/src/SqlLab/components/ShareSqlLabQuery.jsx

```diff
@@ -45,9 +47,19 @@ class ShareSqlLabQuery extends React.Component {
       shortUrl: t('Loading ...'),
     };
     this.getCopyUrl = this.getCopyUrl.bind(this);
+    this.getCopyUrlForSavedQuery = this.getCopyUrlForSavedQuery.bind(this);
+    this.getCopyUrlForKvStore = this.getCopyUrlForKvStore.bind(this);
```

Comment:
`.bind(this)` is only necessary when you are going to be passing `this.getCopyUrlForKvStore` around as a separate variable. If it's only called via `this.getCopyUrlForKvStore` then `this` will be bound correctly.

Reply:
I'll try removing them - I had to add them at one point to get things functioning.